app/partials/wizardNav.html

```diff
@@ -2,8 +2,7 @@
     <ol>
         <li ng-repeat="step in steps" class="step {{step.stepClass}}">
             <span class="{{step.badgeClass}}">{{step.badgeText}}</span>
-            <a class="step-title" ng-if="step.isComplete()" ng-href="/#/{{step.view}}">{{step.title}}</a>
-            <span class="step-title" ng-if="!step.isComplete()">{{step.title}}</span>
+            <span class="step-title" ng-click="setFocused(step)">{{step.title}}</span>
```

Comment:
While you can add an ng-click to a span, for accessibility we need to be presenting the user with an anchor. There are ways around that to continue using a span, but there's not reason why we can't optionally display the anchor here.